dali/test/python/test_operator_brightness_contrast.py

```diff
@@ -134,14 +134,15 @@ def test_equivalence():
     for inp_dtype in [types.FLOAT, types.INT16, types.UINT8]:
       for out_dtype in [types.FLOAT, types.INT16, types.UINT8]:
         for op in ['brightness', 'contrast']:
-          yield check_equivalence, device, inp_dtype, out_dtype, op
+          is_video = random.choice([True, False])
```

Comment:
consider using also "volumetric" inputs (different layout)

Reply:
Renamed it to `has_3_dims`. This test doesn't use layout so I'm not sure if we need to do more.